### PR TITLE
Update user model to include columns from staff table

### LINKED ISSUE #7: Update user table to follow columns in staff table and user model
Update user table to follow columns in [staff table](https://docs.google.com/document/d/1R8RDjLPZR9v0pEUbCnwPrazXRILCwkhhfqVTqwEFvTQ/edit) and [user model](https://docs.google.com/document/d/1R8RDjLPZR9v0pEUbCnwPrazXRILCwkhhfqVTqwEFvTQ/edit) accordingly.

### PR DESCRIPTION
**Pull request by MarcusTXK**
### Changes:
* Update user model to follow columns in [staff table](https://docs.google.com/document/d/1R8RDjLPZR9v0pEUbCnwPrazXRILCwkhhfqVTqwEFvTQ/edit) 
* Add migration file
* Fix eslint warnings for `UserService.ts` and `container.ts`

### Issues:
* Resolves #7

### CODE REVIEW

File: src/models/User.ts [hunk 1]
```diff
@@ -32,15 +47,58 @@ class User extends Model<UserAttributes, UserCreationAttributes> implements User
           type: DataTypes.INTEGER.UNSIGNED,
           autoIncrement: true,
           primaryKey: true,
+          allowNull: false,
```
[the remark this answers is not in the document]
Comment (MarcusTXK) [reply src/models/User.ts:50]: Ah okay I see, will remove this

File: src/db/migrations/20211009035923-add-staff-fields-to-user.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return Promise.all([
+      await queryInterface.addColumn('users', 'staffId', {
```
[the remark this answers is not in the document]
Comment (MarcusTXK) [reply src/db/migrations/20211009035923-add-staff-fields-to-user.js:6]: Okay got it, I thought they might already have existing `staffId` that would have been different from the id of the user

File: src/models/User.ts [hunk 3]
```diff
@@ -5,6 +5,13 @@ interface UserAttributes {
   id: number;
   email: string;
   password: string;
+  staffId?: string;
+  defaultBusId?: string;
+  name?: string;
+  occupation?: string;
```
[the remark this answers is not in the document]
Comment (oliviajohansen) [reply src/models/User.ts:11]: my bad I meant to write `'admin' | 'nurse' | 'driver'` bitwise or | vs logical or ||

**Comment by oliviajohansen** [comment]
also after you merged with the main branch please update the user seed file with new mock data.

**Review by oliviajohansen — APPROVED** [review]
lgtm

File: src/db/migrations/20211009035923-add-staff-fields-to-user.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return Promise.all([
+      await queryInterface.addColumn('users', 'name', {
+        type: Sequelize.STRING,
+        allowNull: false,
+      }),
+      await queryInterface.addColumn('users', 'defaultBusId', {
+        type: Sequelize.STRING,
```
Comment (oliviajohansen) [inline src/db/migrations/20211009035923-add-staff-fields-to-user.js:11]: change type to integer since this id will be taken from the bus table

**Pull request merged**